packages/forms/src/text/FormikCurrencyField.tsx

```diff
@@ -46,15 +48,17 @@ export const FormikCurrencyField: ForwardRefExoticComponent<FormikCurrencyFieldP
         name,
         validate,
       });
-      const errorText = touched && error && formatError(error);
+      const hasError = touched && !!error;
+      const errorText =
+        touched && error !== EMPTY_ERROR_MESSAGE && error && formatError(error);
```

Comment:
Could other components need this behavior? If so please add them before hand in this PR

Reply:
@doniyor2109 I would need to cover the components with tests. Let's do it on the next PR.